frontends/ol-search-ui/src/components/LearningResourceCard.tsx

```diff
@@ -156,7 +156,8 @@ const CardImage: React.FC<CardImageProps> = ({
 const variantClasses: Record<CardVariant, string> = {
   column:        "ol-lrc-col",
   row:           "ol-lrc-row",
-  "row-reverse": "ol-lrc-row-reverse"
+  "row-reverse": "ol-lrc-row-reverse",
+  simplified:    "ol-lrc-simplified"
```

Comment:
**Question:** Here, simplified means "no date/certificate info". Is that correct?

I think we should double check that the absence of that info is actually desirable... It might just be because the designs only use OCW cards, and OCW cards do not currently show dates. *Which might be a bug? I am unsure. #3872*.